run-nitro-enclaves-acm-tests

```diff
@@ -62,7 +62,7 @@ EPS=1
 VERBOSE=0
 # Wait a bit after the NE ACM service has been stopped in order to
 # allow the p11 enclave to terminate
-SLEEP_AFTER_SERVICE_STOP=5
+SLEEP_AFTER_SERVICE_STOP=10
```

Comment:
Given we have to wait for the service to stop the running enclave, which might vary in memory size in the future, we ll have to constantly come to this value and modify accordingly. Would it make sense in a future PR to close this gap and wait in a more deterministic way (e.g. nitro-cli describe polling)?

Reply:
Sure, consulting the output of describe-enclaves should be definitely better (however 10 seconds are more than enough for now, given the p11ne enclave size).